homeassistant/components/rest_command/__init__.py

```diff
@@ -140,18 +146,32 @@ async def async_service_handler(service: ServiceCall) -> None:
                             payload,
                         )
 
+                    _content = None
```

Comment:
Seems best to rely on the suggested content type i.s.o. trying to represent anything, but I'd like a second opinion on this.